docs/kpod-commit.1.md

```diff
@@ -0,0 +1,95 @@
+image can be set using the **author** flag. Various image instructions can be 
+configured with the **--change** flag and a commit message can be set using the
+**--message** flag. The container is paused by default when committing, however
+that can be changed by setting the **pause** flag.
+
+**kpod [GLOBAL OPTIONS]**
+
+**kpod commit [GLOBAL OPTIONS]**
+
+**kpod commit [OPTIONS] CONTAINER**
+
+## OPTIONS
+
+**--author, -a**
+Set the author for the committed image
+
+**--change, -c**
+Apply the following possible instructions to the created image:
+**CMD** | **ENTRYPOINT** | **ENV** | **EXPOSE** | **LABEL** | **STOPSIGNAL** | **USER** | **VOLUME** | **WORKDIR**
+Can be set multiple times
+
+**--message, -m**
+Set commit message for committed image
+
+**--pause, -p**
+Pause the container when creating an image
+
+## EXAMPLES
+
+```
+# kpod commit --change CMD=/bin/bash --change ENTRYPOINT=/bin/sh --change LABEL=blue=image reverent_golick image-commited
+Getting image source signatures
+Copying blob sha256:b41deda5a2feb1f03a5c1bb38c598cbc12c9ccd675f438edc6acd815f7585b86
+ 25.80 MB / 25.80 MB [======================================================] 0s
+Copying config sha256:c16a6d30f3782288ec4e7521c754acc29d37155629cb39149756f486dae2d4cd
+ 448 B / 448 B [============================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+```
+# kpod commit --message "commiting container to image" reverent_golick image-commited
+Getting image source signatures
+Copying blob sha256:b41deda5a2feb1f03a5c1bb38c598cbc12c9ccd675f438edc6acd815f7585b86
+ 25.80 MB / 25.80 MB [======================================================] 0s
+Copying config sha256:af376cdda5c0ac1d9592bf56567253d203f8de6a8edf356c683a645d75221540
+ 376 B / 376 B [============================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+```
+# kpod commit --author "firstName lastName" reverent_golick
+Getting image source signatures
+Copying blob sha256:b41deda5a2feb1f03a5c1bb38c598cbc12c9ccd675f438edc6acd815f7585b86
+ 25.80 MB / 25.80 MB [======================================================] 0s
+Copying config sha256:d61387b4d5edf65edee5353e2340783703074ffeaaac529cde97a8357eea7645
+ 378 B / 378 B [============================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+```
+# kpod commit --pause=false reverent_golick image-commited
+Getting image source signatures
+Copying blob sha256:b41deda5a2feb1f03a5c1bb38c598cbc12c9ccd675f438edc6acd815f7585b86
+ 25.80 MB / 25.80 MB [======================================================] 0s
+Copying config sha256:5813fe8a3b18696089fd09957a12e88bda43dc1745b5240879ffffe93240d29a
+ 419 B / 419 B [============================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+## SEE ALSO
+kpod(1), crio(8), crio.conf(5)
```

Comment:
Remove reference to crio.  Probably link kpod run

Reply:
fixed